config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'user/', to: 'users#index'
+  get 'user/:id', to: 'users#show'
+  get 'user/:id/posts', to: 'users#show'
+  get 'user/:id/posts/:id', to: 'users#show'
```

Comment:
- [ ] Good job following the previous reviewer's advice, but there's an error with your routes, at the moment three of them are pointing to the same `users#show` action. Please consider fixing this! 🤓 🔧 

- [ ] Also, as the previous reviewer pointed out, by convention, routes should be named in plural because they refer to the respective controller, instead of having `user/`, `user/:id`... etc, please name them in their plural form: `users/`, `users/:id`. 🤓 🔧 

- [ ] **[OPTIONAL]** I strongly suggest that you use Rails' helper `resources` as it already handles all of this for you (as long as you follow the correct naming conventions). 🤓 🔧 

```suggestion
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```